### ISSUE TITLE
IFC file containing "$" in header fails to load

### ISSUE THREAD

**Issue by jenslind** [ThatOpen/engine_components#498]
### Describe the bug 📝

I have an IFC model containing the following header:
```
FILE_DESCRIPTION($,'2;1');
```
which fails to load with the following error being thrown:
```
Uncaught (in promise) TypeError: Cannot read properties of null (reading '0')
at .getDescriptionInfo (@thatopen_components.js?v=749602ea:8678:15)
at .getAllGeometries (@thatopen_components.js?v=749602ea:9009:25)
at .load (@thatopen_components.js?v=749602ea:8942:26)
```
It looks like the same issue discussed in https://github.com/ThatOpen/engine_components/issues/207

### System Info 💻

```shell
System:
    OS: macOS 14.6.1
    CPU: (11) arm64 Apple M3 Pro
    Memory: 97.08 MB / 18.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.11.1 - /usr/local/bin/node
    Yarn: 1.22.21 - /usr/local/bin/yarn
    npm: 10.2.4 - /usr/local/bin/npm
  Browsers:
    Chrome: 128.0.6613.138
    Edge: 128.0.2739.79
    Safari: 17.6
```

### Used Package Manager 📦

npm

### Validations ✅

- [X] Read the [docs](https://docs.thatopen.com/intro).
- [X] Check that there isn't [already an issue](https://github.com/ThatOpen/engine_components/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a repository issue and not a framework-specific issue. For example, if it's a THREE.js related bug, it should likely be reported to [mrdoob/threejs](https://github.com/mrdoob/three.js) instead.
- [X] Check that this is a concrete bug. For Q&A join our [Community](https://people.thatopen.com/).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by SoldierFace** [comment]
maybe this will help
https://mega.co.nz/#!B31G2LaZ!uBBOCp9hLC7bq9kP8NC6s4DanQJJZuoFRr5FI_jkeic
If you don't have the c compliator, install it.(gcc or clang)

**Comment by helenkwok** [comment]
I got it! I have messaged @HoyosJuan

**Comment by agviegas** [comment]
Fixed from `@thatopen/components@2.3.5` thanks to @helenkwok!
